our-brothers-web/src/app/shared/components/meetings/meetings-list/meetings-list.component.scss

```diff
@@ -14,6 +18,15 @@
   .join-col {
     flex: 0 1 130px;
   }
+
+  .col {
```

Comment:
Did you test it with populated meetings?

Reply:
no